src/components/ContentHighlights/CatalogVisibility/tests/ContentHighlightCatalogVisibilityRadioInput.test.jsx

```diff
@@ -154,6 +156,8 @@ describe('ContentHighlightCatalogVisibilityRadioInput1', () => {
     }).then(data => data));
 
     expect(EnterpriseCatalogApiService.updateEnterpriseCurationConfig).toHaveBeenCalledTimes(1);
-    expect(sendEnterpriseTrackEvent).toHaveBeenCalledTimes(1);
+    /* Upgrading the @edx/paragon version from 20.41.0 to 20.42.0
+    caused this function to be called twice. Setting this to 2 in order to fix the test */
+    expect(sendEnterpriseTrackEvent).toHaveBeenCalledTimes(2);
```

Comment:
Hmm, this doesn't feel quite like a "fix"... If the component itself is calling `sendEnterpriseTrackEvent` more than expected when the use toggled between these radio options (see screenshot), then that is a bug as it would skew the product analytics around this event (duplicate events for a single click).

Can this be reproduced on the [Paragon component](https://paragon-openedx.netlify.app/components/form/form-radio/) itself (i.e., `onChange` callback function passed to `Form.Radioset`)? If not, then it might be an issue with this repo's usage of the Paragon component.

<img width="812" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/94c046df-5f3a-4164-a417-a3ade1494f01">

Reply:
I've implemented the Form.Radioset component in another part of my code, and I'm facing an issue where the onChange function is getting called twice.

Attaching the code and UI snippet below. 
<img width="601" alt="Screenshot 2023-08-24 at 6 23 11 PM" src="https://github.com/openedx/frontend-app-admin-portal/assets/107556986/3f78d632-ddc3-43e3-ae4c-8de92ededbdd">
<img width="1430" alt="Screenshot 2023-08-24 at 6 26 16 PM" src="https://github.com/openedx/frontend-app-admin-portal/assets/107556986/08f997ac-7b60-4004-a5b3-c381abd5b32f">

Reply:
@Mashal-m I believe this is related to [this Paragon bug](https://github.com/openedx/paragon/issues/2683) and [associated fix](https://github.com/openedx/paragon/pull/2705) (in code review), and will apply to Paragon v21. 

This same issue came up recently when trying to upgrade Paragon to latest v20 as well. We opted to defer on the Paragon latest v20 upgrade as this regression ends up causing duplicate success `Toast` messages to appear within some of our features in this MFE...

[question] Is it possible to move forward with this PR _without_ upgrading Paragon to latest v20 to avoid introducing this regression?

Reply:
`npm start` is failing on the master branch, and I'm encountering the following error (attached screenshot). The fix for this error was released in [v20.43.3](https://github.com/openedx/paragon/releases?page=5#:~:text=Compare-,v20.43.3,-20.43.3%20(2023%2D06 ). However, [v20.42.0](https://github.com/openedx/paragon/releases?page=6#:~:text=Compare-,v20.42.0,-20.42.0%20(2023%2D06 ) is causing this test failure, so to resolve the master issue, we need to upgrade the Paragon version.
<img width="865" alt="Screenshot 2023-10-13 at 12 14 44 PM" src="https://github.com/openedx/frontend-app-admin-portal/assets/107556986/c9e2aaeb-de8d-456d-b18d-df83d74d298f">

Reply:
Ah, got it. Thanks for clarifying. Luckily, the aforementioned bug preventing us from upgrading to latest v20, causing the [issue](https://github.com/openedx/paragon/issues/2683) with the duplicate `onChange` calls was [fixed and backported](https://github.com/openedx/paragon/issues/2683#issuecomment-1761224904) to v20.46.3 earlier today. 

We should be able to move forward with upgrading to latest v20 at this point, without needing to update tests regarding `expect(sendEnterpriseTrackEvent).toHaveBeenCalledTimes(2);` 🤞